src/_shared/scss/_adverts.scss

```diff
@@ -569,12 +569,16 @@
     background: #eaeaea;
 
     > .adverts__header .button {
-        color: #ffffff;
+        color: #000000;
     }
 
     .adverts__blurb {
         font: normal 14px/18px $f-serif-headline;
-        color: #ffffff;
+        margin-top: 20px;
+        color: #000000;
+        background: #ffffff;
+        -webkit-box-decoration-break: clone;
```

Comment:
Just use `box-decoration-break` here; the transpiler will add all of the necessary variants when it builds the templates.

Reply:
I'll change that - sorry, you told me that already...